front/src/applications/stdcmV2/components/StdcmInputVia.tsx

```diff
@@ -9,39 +9,44 @@ import { ISO8601Duration2sec } from 'utils/timeManipulation';
 
 import { StdcmStopTypes } from '../types';
 
+type StdcmInputViaProps = {
+  stopType?: StdcmStopTypes;
+  pathStepStopFor: PathStep['stopFor'];
+  updatePathStepStopTime: (stopTime: string) => void;
+};
+
 const StdcmInputVia = ({
   stopType,
-  pathStep,
+  pathStepStopFor,
```

Comment:
As we don't use `pathStepStopType` for the other prop

```suggestion
  stopDuration,
```